File: PRAK305-2310817210001-AndraBraputraAkbarSaleh.py
s = int(input())
d = s // 86400  
s = s - d * 86400
a = s // 3600  
s = s - a * 3600
b = s // 60
s = s - b * 60
if d == 0:
    print(f"{a:02d}:{b:02d}:{s:02d}")
else:
    print(f"{d} hari {a:02d}:{b:02d}:{s:02d}")

File: PRAK305-2310817210001-AndraBraputraAkbarSaleh.c
#include <stdio.h>

int main() {
    int s, d, a, b;
    scanf("%d", &s);
    d = s / 86400;
    s = s - d * 86400;
    a = s / 3600;
    s = s - a * 3600;
    b = s / 60;
    s = s - b * 60;

    if (d == 0) {
        printf("%02d:%02d:%02d\n", a, b, s);
    } else {
        printf("%d hari %02d:%02d:%02d\n", d, a, b, s);
    }
    return 0;
}

   